### PR TITLE
Updated the translation/* articles to Symfony 4

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

**Comment by yceruto** [comment]
Could you to update the framework reference to add the default path option? https://github.com/symfony/symfony-docs/pull/8634/files#diff-c0c158a40398b72fa4f0c9d0accbc169

**Comment by javiereguiluz** [comment]
@yceruto sorry but I can't. I don't see the default_path in my branch or in 4.0. Is this only on master?

**Comment by yceruto** [comment]
@javiereguiluz are you asking by Symfony repo or docs? I don't understand the question :/

https://github.com/symfony/symfony/blob/3.4/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L680

https://github.com/symfony/symfony-docs/pull/8634/files#diff-c0c158a40398b72fa4f0c9d0accbc169 It needs to be added to the framework reference?

**Comment by javiereguiluz** [comment]
yes, I was referring to the docs ... but I'm probably wrong.

File: translation.rst [hunk 1]
```diff
@@ -48,24 +48,37 @@ to learn even more. Overall, the process has several steps:
 
 .. _translation-configuration:
 
+Installation
+------------
+
+In applications using :doc:`Symfony Flex </setup/flex>`, run this command to
+install the translator before using it:
```
Comment (weaverryan) [inline translation.rst]: I'm not mentioning this in 4.0: I'm assuming they have Flex installed and have that dir structure. If we don't assume this, then the docs become a mess.

File: translation.rst [hunk 2]
```diff
@@ -48,24 +48,37 @@ to learn even more. Overall, the process has several steps:
...
+
+    $ composer require translator
+
 Configuration
 -------------
 
-Translations are handled by a ``translator`` service that uses the user's
-locale to lookup and return translated messages. Before using it, enable the
-``translator`` in your configuration:
+The previous creates an initial config file where you can define the default
```
Comment (weaverryan) [inline translation.rst]: > The previous command...

File: translation.rst [hunk 3]
```diff
@@ -48,24 +48,37 @@ to learn even more. Overall, the process has several steps:
...
+locale of the app and the :ref:`fallback locales <translation-fallback>` used
```
Comment (weaverryan) [inline translation.rst]: > ... fallback locales that will be used if Symfony...

**Comment by weaverryan** [comment]
@javiereguiluz He already did the work for you :). @yceruto just wants you to import his work from another PR: https://github.com/symfony/symfony-docs/pull/8634/files#diff-c0c158a40398b72fa4f0c9d0accbc169

**Review by weaverryan — CHANGES_REQUESTED** [review]
Minor changes, then 👍 from me

**Comment by weaverryan** [comment]
Status: Needs work

**Pull request merged**